src/engine/Service.cpp

```diff
@@ -384,3 +362,44 @@ ProtoResult Service::makeNeutralElementResultForSilentFail() const {
   }
   return {std::move(idTable), resultSortedOn(), LocalVocab{}};
 }
+
+// ____________________________________________________________________________
+void Service::verifyVariables(
+    const nlohmann::json& j,
+    const ad_utility::LazyJsonParser::Generator& gen) const {
```

Comment:
It suffices to pass `const Details&` as the second argument, I was confused what you need the generator for here and how this should work:)